include/drivers/spi.h

```diff
@@ -116,7 +112,21 @@ extern "C" {
  * the CS control to a gpio line through struct spi_cs_control would be
  * the solution.
  */
-#define SPI_CS_ACTIVE_HIGH	BIT(15)
+#define SPI_CS_ACTIVE_HIGH	BIT(14)
+
+/**
+ * @brief SPI MISO lines (if only CONFIG_SPI_EXTENDED_MODES is enabled)
```

Comment:
nit: this will just document first define, not the others. You can use

```c
/**
 * @name SPI MISO lines ....
 * @{
 */

...

/** @} */
```